zajecia-3/gulpfile.js

```diff
@@ -16,7 +18,9 @@ const path = {
 
 const css = function() {
     return gulp.src(path.cssSrc)
-        .pipe(sass().on("error", sass.logError))
+        .pipe(sass({
+            outputStyle: 'compressed'
```

Comment:
lepiej bylo by stworzyc nowy task, bo teraz modyfikujac glowny task CSS, wlaczyles kompresje zarowno w wersji production, jak i tej developerskiej, ciezej sie to potem debuguje